vms/evm/contract_message_test.go

```diff
@@ -59,15 +53,15 @@ func TestUnpack(t *testing.T) {
 	}{
 		{
 			name:          "valid",
-			input:         "0000000000007fc93d85c6d62c5b2ac0b519c87010ea5294012d1e407030d6acd0021cac10d50000005200000000000027ae10273d17cd7e80de8580a51f476960626e5f0000000000000000000000000000000000000000000000000000000000000000123412341234123412341234123412341234123400000000",
-			networkID:     0,
+			input:         "0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000024c00000000053968786a235cbcfb6e57321b94378e95939b773a9626acf7a8cc440075c02c7268000002220000000000010000001452718d4ea91a6dd9a68940dbd687efa32315d11600000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000008db97c7cece249c2b98bdc0226cc4c2a57bf52fcb1d32d469938520383696931c26b9753662db74ad33c012f41e337aa828f1b74000000000000000000000000abcedf1234abcedf1234abcedf1234abcedf12340000000000000000000000000000000000000000000000000000000000002710000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000180000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001000000000000000000000000a100ff48a37cab9f87c8b5da933da46ea1a5fb80000000000000000000000000000000000000000000000000000000000000002acafebabecafebabecafebabecafebabecafebabecafebabecafebabecafebabecafebabecafebabecafe000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+			networkID:     0x539,
```

Comment:
Can you add a comment explaining what this network ID is?

Reply:
I got the data from the integration test. When we did integration test, we sent cross chain message. The data is from the log event of sendCrossChain message. Then unpack the data, we can get [unsigned message](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/unsigned_message.go#L15) that has networkID.

Reply:
Gotcha. Decoding the hex, this is `1337`, the network ID used by avalanche-cli for local networks. This is defined already in the variable [LocalNetworkID](https://github.com/ava-labs/awm-relayer/blob/main/peers/app_request_network.go#L27). Let's just use that directly here.

Reply:
cool, fixed to use LocalNetworkID